networking-cisco/networking-cisco.sav/rpm/build-rpm.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
what is this "networking-cisco.sav" directory for? Is it in the commit by mistake?